### PR TITLE
[DECLUTTER USING] remove lost password strategy

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch removes the lost password strategy chapter. It was an article by Adam, and explains some design rational, but it has nothing to do with `using` wasabi, and does not provide any actionable user help. 

I think the password finder chapter is enough.

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
Why @RiccardoMasutti ?
Can you please provide rational and arguments?

**Comment by RiccardoMasutti** [comment]
Because we should optimize the documentation for newbies, not for experts.

I think such a chapter should remain.

**Comment by MaxHillebrand** [comment]
> Moving Password Confirmation to Next Page

This is a rational for `wallet developers`, not for wallet users

> Password Tester

This is a rational for `wallet developers`, not for wallet users

> Password Finder

This is for users, but it has a standalone chapter.

**Comment by RiccardoMasutti** [comment]
> > Moving Password Confirmation to Next Page
> 
> This is a rational for `wallet developers`, not for wallet users
> 
> > Password Tester
> 
> This is a rational for `wallet developers`, not for wallet users
> 
> > Password Finder
> 
> This is for users, but it has a standalone chapter.

Ok ty

@yahiheb thoughs?

File: docs/using-wasabi/10Commandments.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ It also gives you time to move your coins to a different wallet if you notice th
 Wasabi is a fully non-custodial wallet, which means that you are always in possession of your keys, and this means safely storing a backup.
 If you have done this step, then even if your computer crashes, there is very little you have to worry about, you can always [recover your wallet](/using-wasabi/WalletRecovery.md).
 Also, under no circumstance should you reveal the password or mnemonic words to anyone that asks for them.
-Lastly, understand that if you [lose your password](/using-wasabi/LostPassword.md), it becomes much harder (if not impossible) to restore your wallet - so store both safely!
+Lastly, understand that if you lose your password, it becomes much harder (if not impossible) to restore your wallet - so store both safely!
```
Comment (yahiheb) [inline docs/using-wasabi/10Commandments.md:34]:
```suggestion
Lastly, understand that if you lose your password, it becomes much harder (if not impossible) to restore your wallet, so store both safely!
```

**Comment by yahiheb** [comment]
I think I ACK removing this.
IMO the whole password test process should change when generating a wallet.

**Pull request merged**